crud/insert.lua

```diff
@@ -121,6 +122,13 @@ local function call_insert_on_router(vshard_router, space_name, original_tuple,
         end
 
         return nil, err_wrapped
+    else
+        if schema.result_needs_reload(space, storage_result) then
```

Comment:
Did I understand the idea correctly?

```lua
local err_wrapped = InsertError:new("Failed to insert: %s", storage_result.err)

if schema.result_needs_reload(space, storage_result) then
    return nil, err_wrapped, const.NEED_SCHEMA_RELOAD
end

if storage_result.err ~= nil then
    return nil, err_wrapped
end
```

Code above causes a duplicate key error, since, as I understand it, the `insert` triggered several times in `schema.wrap_func_reload`

Reply:
> Code above causes a duplicate key error

Let's check schema before actual storage operation